blog.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="zxx">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="styles/main-style.css">
+        <link rel="stylesheet" media="screen and (max-width: 500px)" href="styles/width-small.css">
+        <link rel="stylesheet" media="screen and (min-width: 501px) and (max-width: 850px)" href="styles/width-medium.css">
+        <link rel="stylesheet" media="screen and (min-width: 851px)" href="styles/width-large.css">
+        <title>Dog Blog</title>
+    </head>
+    <body>
+            <header class="header-container">
+                    <div class="header-item-1"><img src="images/cart-icon.svg" alt="cart icon">
```

Comment:
When you wrap things in divs make sure to do it like this

<div>
  <h1>Hi there</h1>
</div>

Also you probably don't need all of these divs in here.